packages/ui-extensions/src/surfaces/point-of-sale/components/shared/InputField.ts

```diff
@@ -29,17 +38,56 @@ export interface InputAction {
  * @property {number?} [maxLength] - Specifies the maximum number of characters allowed.
  */
 export interface InputProps {
+  /**
+   * Whether the field can be modified.
+   */
   disabled?: boolean;
+  /**
+   * Indicates an error to the user. The field is given specific stylistic treatment to communicate problems that have to be resolved immediately.
+   */
   error?: string;
+  /**
+   * The content to use as the field label.
+   */
   label: string;
+  /**
+   * The callback when focus is removed.
+   */
   onBlur?: () => void;
+  /**
+   * The callback when the user has finished editing a field.
+   */
   onChange?: (value: string) => void;
+  /**
+   * The callback when input is focused.
+   */
   onFocus?: () => void;
+  /**
+   * Callback when the user makes any changes in the field. As noted in the documentation for `onChange`, you **must not** use this to update `value` — use the `onChange` callback for that purpose. Use the `onInput` prop when you need to do something as soon as the user makes a change, like clearing validation errors that apply to the field as soon as the user begins making the necessary adjustments.
+   */
   onInput?: (value: string) => void;
+  /**
+   * A short hint that describes the expected value of the field.
+   */
   placeholder?: string;
+  /**
+   * Whether the field needs a value.
+   */
   required?: boolean;
+  /**
+   * The current value for the field. Defaults to now. You should update this value in response to the `onChange` callback.
```

Comment:
This is generic to InputField so it does not necessarily default to Now. That's for Date/Time only! Probably just a copy/paste oversight